tests/src/JIT/HardwareIntrinsics/X86/Avx2/ConvertToVector256Int16.Byte.cs

```diff
@@ -28,6 +29,9 @@ private static void ConvertToVector256Int16Byte()
                 // Validates basic functionality works, using Unsafe.Read
                 test.RunBasicScenario_UnsafeRead();
 
+                // Validates basic functionality works, using the pointer overload
+                test.RunBasicScenario_Ptr();
```

Comment:
These tests in particular are written as if they are templated but aren't actually generated by a template right now. It would be good to just move these to be templated in a follow up PR.

For the time being, I've manually updated each one to include a test that covers the overload that takes a pointer (for both regular and reflection calls).

Reply:
Could you file an issue to update the test generator?

Reply:
I think we already have one, but I will double check,

Reply:
I think the issue is sufficiently covered by https://github.com/dotnet/coreclr/issues/19246, which is tracking all remaining tests being moved to be templated.

Reply:
Great, thanks